service/keys/keys.go

```diff
@@ -51,3 +88,8 @@ func (keys *keys) GetNostrPublicKey() string {
 func (keys *keys) GetNostrSecretKey() string {
 	return keys.nostrSecretKey
 }
+
+func (keys *keys) EncryptChannelBackupData(channelBackupData string) (string, error) {
+	// FIXME: this is not the right way to encrypt the data (we already have a key, no need to generate a new one)
+	return config.AesGcmEncrypt(channelBackupData, keys.encryptedChannelsBackupKey)
```

Comment:
I think this should not be used there (or needs refactoring) the key should not go through the `DeriveKey` function which creates a key from a password string using argon2. 

we already have a key here, don't we?
or is that how one generally does this?